### DISCUSSION TITLE
Taranis X9D+2019 - No other language, only English?

### DISCUSSION THREAD

**Discussion by rafpigna** [EdgeTX/edgetx#3503] [Q&A]
This seems strange to me. I cannot find a firmware version in other language than EN. It's right? 
I was looking for italian and I see that the X9D+ (not 2019 version) has it, but not the 2019 version. I think that should be quite similar. 
How can I create a translation for this model? Some one willing to exlpain? I'm not a developer but I can translate it. 

Thanks

**Comment by pfeerick** [comment]
I guess you're referring to the lang-firmwares 2.8.3 multi-language builds, and didn't notice this in the release notes there?
>  and unfortunately all but EN X9D+2019 firmware is also now too big to compile with standard options.


Unfortunately, they are similar, but not the same. i.e. X9D+2019 as ISRM, not ACCST RF, as well as few other minor things. I tried to build for all of the languages, but for 2.8.3 none of the multi-language builds were successful (they were bigger than the flash memory of the radio). However I think IT was successful for both 2.8.2 and 2.8.1
